.github/workflows/scan.yaml

```
@@ -5,6 +5,10 @@ on:
         required: true
       GITHUBACTIONSTOKEN:
         required: true
+    inputs:
+      submodules:
+        required: false
```

I noticed (while testing this out) that there are some other required fields for inputs with workflow_call https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs

```suggestion
        description: 'Enable submodule (recursive) checkout by actions/checkout'
        # Allowed values are "" or "false" (default), "true" and "recursive" - see https://github.com/actions/checkout/blob/main/README.md#usage
        type: string
        required: false
```